### ISSUE TITLE
Parallel coordinates plot

### ISSUE THREAD

**Issue by Wycklife** [africanmathsinitiative/R-Instat#4508]
When working on parallel coordinates plots dialog , in the subdialog we can only order in three ways **(Any class, All class , Skewness)** ,yet when working on the graphics we needed to order by a column as shown in the code below.Could a selector and a receiver for the orders be added so that this is possible?

![snipping](https://user-images.githubusercontent.com/33898404/38547078-f3b19aec-3cb6-11e8-85ef-be7cd5b6c374.PNG)

**Comment by dannyparsons** [comment]
Good question - @maxwellfundi had you looked into this when you did the dialog?

**Comment by maxwellfundi** [comment]
@dannyparsons  Not really. This has come up from the graphs that are being generated from which had not been considered in the before.

**Comment by dannyparsons** [comment]
I think this should be added then. When depends on how urgent it is @rdstern

**Comment by maxwellfundi** [comment]
When we were doing this with @shadrackkibet  and @ZIRANA11  we figured out this would be a good addition to enhance functionality of the dialog.

**Comment by dannyparsons** [comment]
Great, if it's quick then I suggest just do it.

**Comment by maxwellfundi** [comment]
It needs careful thoughts because there are other additions on the subdialog I think might be important to add. Example is in the order, there is long list of possible things to add but from the previous issue @rdstern  advised to have the current ones. So lets hear what @rdstern  thinks and discuss how to implement that possibly inclusive of other features.
